diff -Nur opentumblr-0.0.2/setup.py opentumblr-0.0.2.new/setup.py
--- opentumblr-0.0.2/setup.py	2009-05-05 16:28:19.000000000 -0500
+++ opentumblr-0.0.2.new/setup.py	2009-05-05 17:04:33.000000000 -0500
@@ -24,13 +24,6 @@
 	doc_files =['AUTHORS','INSTALL','LICENSE','README','THANKS']
 	icon_files = ['opentumblr/images/opentumblr.png','opentumblr/images/opentumblr.xpm']
 	
-	if not os.path.isdir(ipath_docs):
-		os.mkdir(ipath_docs)
-	
-	if not os.path.isdir(ipath_images):
-		os.mkdir(ipath_images)
-		os.mkdir(ipath_dashboard)
-	
 	setup(
 		name=NAME,
 		version=VERSION,
@@ -69,4 +62,4 @@
 	    data_files=DATA_FILES,
 	    options={'py2app': OPTIONS},
 	    setup_requires=['py2app'],
-	)
\ No hay ningún carácter de nueva línea al final del fichero
+	)
